code_samples/action_basics/build.gradle

```diff
@@ -16,8 +16,9 @@ repositories {
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-  version '2019.3.3'
-  updateSinceUntilBuild = false
+  // Uncomment the version line below when the product releases
```

Comment:
we need to have _one_ version active here
also please do not put commented stuff in if avoidable